checkov/helm/runner.py

```diff
@@ -155,10 +155,16 @@ def _convert_chart_to_k8s(self, chart_item):
         target_dir.replace("//", "/")
         chart_name = chart_meta.get('name', chart_meta.get('Name'))
         chart_version = chart_meta.get('version', chart_meta.get('Version'))
-        if target_dir.endswith('/'):
-            target_dir = target_dir[:-1]
-        if target_dir.endswith(chart_name):
-            target_dir = target_dir[:-len(chart_name)]
+        try:
+            if target_dir.endswith('/'):
+                target_dir = target_dir[:-1]
+            if target_dir.endswith(chart_name):
+                target_dir = target_dir[:-len(chart_name)]
```

Comment:
what was the error here? the length of the string must be enough (the `endswith` function should verify it)

Reply:
TypeError: endswith first arg must be str or a tuple of str, not NoneType

Reply:
if `target_dir` is  `None`, then a few lines higher `target_dir.replace("//", "/")` should also fail.
probably just do a `None` check after line 154.
```python
target_dir = chart_dir.replace(self.root_folder, f'{self.target_folder_path}/')
if target_dir is None:
    raise ...
```

Reply:
@gruebel but it didn't fail on a few lines higher, it fail on `if target_dir.endswith(chart_name):`

Reply:
ah got it, the problem is the `chart_name`, it could be `None`. This should work on line 156
```python
chart_name = chart_meta.get('name', chart_meta.get('Name'))
if chart_name is None:
    raise ...
```